### ISSUE TITLE
CDDA Game Launcher Crash

### ISSUE THREAD

**Issue by Wolfnation21** [DazedNConfused-/CDDA-Game-Launcher#51]
Went to open Cataclysm launcher to play and received a box saying something went wrong with the full message being "the CDDA game launcher just crashed. An unhandled exception was raised."
* Version: 1.6.5
* OS: Windows-10-10.0.19044-SP0 (64-bit)
* Type: `<class 'IndexError'>`
* Value: list index out of range
* Traceback:
```
  File "C:\Users\13129\Documents\CDDA Game Launcher\cddagl\ui\views\main.py", line 1379, in showEvent
    self.refresh_builds()
  File "C:\Users\13129\Documents\CDDA Game Launcher\cddagl\ui\views\main.py", line 3235, in refresh_builds
    self.refresh_changelog()
  File "C:\Users\13129\Documents\CDDA Game Launcher\cddagl\ui\views\main.py", line 3274, in refresh_changelog
    msg = body[3]

```

**Comment by isloat** [comment]
I get the same exception and crash as well.

**Comment by volnn** [comment]
I also get the same exception and crash, exactly the same.

**Comment by SirClarke** [comment]
https://github.com/DazedNConfused-/CDDA-Game-Launcher/issues/49#issuecomment-1140114882

this person has a temporary fix

**Comment by DazedNConfused-** [comment]
Marking this as duplicate of: https://github.com/DazedNConfused-/CDDA-Game-Launcher/issues/49

**Comment by DazedNConfused-** [comment]
Shout-out to #52: should have gotten fixed in v1.6.6

Please let me know if anything else comes up.
